tools/gen_lang_stubs.py

```
@@ -0,0 +1,181 @@
+    parts.append("")
+    parts.append("class _LangAccessorTyped:")
+    parts.append('    """self.lang 的类型化声明（仅类型提示，运行时由 __getattr__ 动态加载）"""')
+    for module_name, _ in modules:
+        cls = pascal_case(module_name) + "Module"
+        parts.append(f"    {module_name}: {cls}")
+    parts.append("")
+
+    TYPED_OUT.write_text("\n".join(parts), encoding="utf-8")
+    print(f"[ok] 已生成 {TYPED_OUT.relative_to(REPO_ROOT)}（{len(modules)} 个模块）")
+
+    # 幂等更新 __init__.py：LangAccessor 继承 _LangAccessorTyped
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)